### DISCUSSION TITLE
Produce a new AC model from performance tables

### DISCUSSION THREAD

**Discussion by nivkuli** [junzis/openap#37] [Q&A]
Hi @junzis,

I understood that the method for adding AC types is with data from OpenSky, which is naturally limited to ADS-B equipped AC only.
Is there an option to have a model made from performance tables to support older non-ADS-B types?
which dataset will be required in that case?

Best regards,

Niv

**Comment by junzis** [comment]
Yes, certainly. I think as long as the data is openly available, we can add it to OpenAP.

Which aircraft types do you have in mind? And would you please also point me out to the performance data sources? Thanks!

**Reply by nivkuli** [reply]
Thanks for the quick reply!
I am looking for a performance model for a B703.
I have a performance manual with cruise tables (LRC, .82M, etc), but still not sure about the license of the data.
Until I'll have an answer - what parameters do I need for the OpenAp model? Do you think it's possible to modulate them from performance tables?

**Reply by junzis** [reply]
Hi @nivkuli, you could refer to the following data structure of A320:

Basic info: https://github.com/junzis/openap/blob/master/openap/data/aircraft/a320.yml
Drag polar: https://github.com/junzis/openap/blob/master/openap/data/dragpolar/a320.yml

Thanks!
